src/main/services/AuthService

```diff
@@ -10,6 +10,12 @@ public class AuthService {
     private final String userFile = "users.txt";  
 
     public boolean authenticate(String username, String password) {
+        //added checks for null or invalid input
+        if (username == null || password == null || username.isEmpty() || password.isEmpty()) {
+            System.err.println("Invalid username or password");
+            return false;
+        }
+
```

Comment:
Nulls were not handled properly in Authservice. I included the null and invalid
checks for username and password.